proto/vendorinfo/tx.proto

```diff
@@ -1,13 +1,47 @@
 syntax = "proto3";
 package zigbeealliance.distributedcomplianceledger.vendorinfo;
 
+import "vendorinfo/vendor_info.proto";
+import "gogoproto/gogo.proto";
+import "cosmos_proto/cosmos.proto";
 // this line is used by starport scaffolding # proto/tx/import
 
 option go_package = "github.com/zigbee-alliance/distributed-compliance-ledger/x/vendorinfo/types";
 
 // Msg defines the Msg service.
 service Msg {
-    // this line is used by starport scaffolding # proto/tx/rpc
+  rpc CreateVendorInfo(MsgCreateVendorInfo) returns (MsgCreateVendorInfoResponse);
+  rpc UpdateVendorInfo(MsgUpdateVendorInfo) returns (MsgUpdateVendorInfoResponse);
+  rpc DeleteVendorInfo(MsgDeleteVendorInfo) returns (MsgDeleteVendorInfoResponse);
+// this line is used by starport scaffolding # proto/tx/rpc
 }
 
+message MsgCreateVendorInfo {
+  string creator = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  uint64 vendorID = 2 [(gogoproto.moretags) = "validate:\"required,gte=0,lte=65535\""];
```

Comment:
Yes, let's change it to `int32`. The problem with `uint64` is that it will be output as a string (not as a number) in JSON outputs (CLI output for example).